dashboard/app/reporting_email.go

```diff
@@ -102,6 +106,115 @@ func (cfg *EmailConfig) Validate() error {
 	return nil
 }
 
+// handleCoverageReports sends reports for the previous period.
```

Comment:
Suggestion:
`// handleCoverageReports sends a coverage report for the two full months preceding the current one.`